README.md

```diff
@@ -275,6 +275,9 @@ func (*User) Indexes() []*myddlmaker.Index {
 
         // INDEX `idx_name` (`name`) INVISIBLE
         myddlmaker.NewIndex("idx_name", "name").Invisible(),
+
+		// INDEX `idx_name`(`id1` ASC, `id2` DESC)
+		myddlmaker.NewIndex("idx_name", "name").ASC("id1").DESC("id2"),
```

Comment:
This example is wrong.
Because the index doesn't have columns `id1` and `id2`, `ASC("id1").DESC("id2")` is ignored.

```suggestion
		// INDEX `idx_name`(`id1` ASC, `id2` DESC)
		myddlmaker.NewIndex("idx", "id1", "id2").ASC("id1").DESC("id2"),
```

Reply:
Oops... Sorry, I had missed a copy and paste.
I fixed it.